src/slack.js

```diff
@@ -48,14 +48,22 @@ const updateSlackStatus = async ({ token, text, emoji }) => {
  * or not the other promises have resolved.
  * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
  */
-module.exports = (isInMeeting = false, workspaces = slackWorkspaces) =>
+module.exports = (eventUserMail = '', isInMeeting = false, workspaces = slackWorkspaces) =>
   axios.all(
     workspaces.map((slackWorkspace) => {
-      const status = isInMeeting ? 'meetingStatus' : 'noMeetingStatus'
-      return updateSlackStatus({
-        token: slackWorkspace.token,
-        text: slackWorkspace[status].text,
-        emoji: slackWorkspace[status].emoji,
-      })
-    }),
-  )
+      logger('userEmail', slackWorkspace.userMail)
+      logger('eventUserMail', eventUserMail)
+      if (slackWorkspace.userMail === undefined || slackWorkspace.userMail === eventUserMail) {
```

Comment:
```suggestion
      if (workspaces.userMail && workspaces.userMail === eventUserMail) {
```

We must use `workspaces` her instead of `slackWorkspaces` to be able to test this fn.

See https://github.com/natterstefan/zoom-slack-status-updater/blob/master/src/__tests__/slack.test.js#L56-L59, for instance.

Reply:
This did not work for me. And I do not understand how it should. The workspaces is an array, whereas the slackWorkspace is one object. And the filter is based on the single workspace.